clients/src/main/java/org/apache/kafka/common/network/KafkaChannel.java

```diff
@@ -24,13 +24,15 @@
 import java.nio.channels.SelectionKey;
 
 import java.security.Principal;
+import java.util.concurrent.atomic.AtomicLong;
 
 import org.apache.kafka.common.utils.Utils;
 
 public class KafkaChannel {
     private final String id;
     private final TransportLayer transportLayer;
     private final Authenticator authenticator;
+    private final AtomicLong networkThreadTimeNanos;
```

Comment:
Because the value is updated on the network thread and read-and-reset on the request handler thread. I have added a comment.